SpatialGDK/Documentation/toc.md

```diff
@@ -33,7 +33,7 @@
  ***
 - <h3>[- Troubleshooting]({{urlRoot}}/content/troubleshooting)</h3>
 - <h3>[- Known issues]({{urlRoot}}/known-issues)</h3>
-- <h3>[- Ability system]({{urlRoot}}/content/ability-system)</h3>
+- <h3>[- Gameplay Ability system]({{urlRoot}}/content/ability-system)</h3>
```

Comment:
Maybe a compromise is to have this page in the reference section and then have the page referenced and linked-to from the  Known issues and/or Troubleshooting pages?

Reply:
Done, see the latest commit. Might want @girayimprobable 's input on having it in the known issues.

Reply:
LGTM